### PR TITLE
feat/savax-vault Adding sAVAX vault

### PR DESCRIPTION
**Pull request by fqlx**
feat/savax-vault Adding sAVAX vault

### CODE REVIEW

File: shared/src/assets/icons/erc20Assets.tsx [hunk 1]
```diff
@@ -306,6 +306,30 @@ export const WAVAXLogo: React.FC<SVGProps & { showBackground?: boolean }> = ({
...
+    viewBox="0 0 56 56"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+    {...props}
+  >
+    {showBackground && (
+      <path
+        d="M28 56C43.464 56 56 43.464 56 28C56 12.536 43.464 0 28 0C12.536 0 0 12.536 0 28C0 43.464 12.536 56 28 56Z"
+        fill="#11BAF0"
+        fill-opacity="0.12"
```
Comment (kenchangh) [inline shared/src/assets/icons/erc20Assets.tsx:323]: We need to change `fill-opacity` to `fillOpacity`

File: shared/src/components/Product/productCopies.tsx [hunk 2]
```diff
@@ -501,6 +501,62 @@ export const productCopies: { [vault in VaultOptions]: ProductCopies } = {
...
+          explanation="A covered call refers to a financial transaction in which the investor selling call options owns an equivalent amount of the underlying security."
+          learnMoreURL="https://www.investopedia.com/terms/c/coveredcall.asp"
+          renderContent={({ ref, ...triggerHandler }) => (
+            <HighlightedText ref={ref} {...triggerHandler}>
+              covered call
+            </HighlightedText>
+          )}
+        />{" "}
+        strategy. The vault reinvests the yield earned back into the strategy,
+        effectively compounding the yields for depositors over time.
```
Comment (kenchangh) [inline shared/src/components/Product/productCopies.tsx:525]: Here we can add the bit about depositing into BENQI's staked avax contract, like with Lido

https://github.com/ribbon-finance/ribbon-frontend/blob/26855b78f56594f55587b2b5e9a251f0bf6561c6/shared/src/components/Product/productCopies.tsx#L332-L353

```js
 <>
        The vault earns yield on its ETH deposits by running a weekly automated
        ETH{" "}
        <TooltipExplanation
          title="COVERED CALL"
          explanation="A covered call refers to a financial transaction in which the investor selling call options owns an equivalent amount of the underlying security."
          learnMoreURL="https://www.investopedia.com/terms/c/coveredcall.asp"
          renderContent={({ ref, ...triggerHandler }) => (
            <HighlightedText ref={ref} {...triggerHandler}>
              covered call
            </HighlightedText>
          )}
        />{" "}
        strategy where it stakes its ETH deposits in{" "}
        <TooltipExplanation
          title="LIDO"
          explanation="Lido empowers stakers to put their staked assets to use. "
          learnMoreURL="https://lido.fi"
          renderContent={({ ref, ...triggerHandler }) => (
            <HighlightedText ref={ref} {...triggerHandler}>
              Lido
            </HighlightedText>
          )}
        />{" "}
        and then uses its{" "}
        <TooltipExplanation
          title="STETH"
          explanation="stETH is the deposit token that represents a user's share of the their ETH on the Ethereum beacon chain."
          learnMoreURL="https://lido.fi/ethereum"
          renderContent={({ ref, ...triggerHandler }) => (
            <HighlightedText ref={ref} {...triggerHandler}>
              stETH
            </HighlightedText>
          )}
        />{" "}
        to collateralize weekly out-of-money ETH call options. The yield earned
        from both the covered call strategy and the ETH staking rewards are
        reinvested weekly, effectively compounding the yields for depositors
        over time.
      </>
```

File: shared/src/assets/icons/erc20Assets.tsx (hunk 1 above)
Comment (fqlx) [reply shared/src/assets/icons/erc20Assets.tsx:323]: Fixed - good catch

File: shared/src/components/Product/productCopies.tsx (hunk 2 above)
Comment (fqlx) [reply shared/src/components/Product/productCopies.tsx]: Done

**Review by stevenwal — APPROVED** [review]
LGTM

**Pull request merged**